scripts/funcTests/runFuncTests.sh

```diff
@@ -55,8 +55,8 @@ fi
 
 if [ "$MONO_TESTS" == "1" ]; then
     # Run mono test
-    TestDir="$DIR/artifacts/NuGet.CommandLine.Test/"
```

Comment:
I would update the yaml to get the files to be in a subdirectory since these files will potentially overwrite actual artifacts

Reply:
Good idea!